fuzz/src/lib.rs

```diff
@@ -0,0 +1,70 @@
+use std::ops::Range;
+
+use libfuzzer_sys::arbitrary::{Arbitrary, Result, Unstructured};
+use vortex::Array;
+use vortex_sampling_compressor::compressors::alp::ALPCompressor;
+use vortex_sampling_compressor::compressors::bitpacked::BitPackedCompressor;
+use vortex_sampling_compressor::compressors::dict::DictCompressor;
+use vortex_sampling_compressor::compressors::r#for::FoRCompressor;
+use vortex_sampling_compressor::compressors::roaring_bool::RoaringBoolCompressor;
+use vortex_sampling_compressor::compressors::roaring_int::RoaringIntCompressor;
+use vortex_sampling_compressor::compressors::runend::DEFAULT_RUN_END_COMPRESSOR;
+use vortex_sampling_compressor::compressors::sparse::SparseCompressor;
+use vortex_sampling_compressor::compressors::zigzag::ZigZagCompressor;
+use vortex_sampling_compressor::compressors::EncodingCompressor;
+
+pub struct FuzzArrayAction {
+    pub array: Array,
+    pub action: Action,
```

Comment:
```suggestion
    pub action: Vec<Action>,
```
?

Reply:
Changing that, but building actually chained operations will probably be more work